src/redux/actions/fetchDataAction.tsx

```diff
@@ -0,0 +1,86 @@
+export const fetchDataActionType = {
+    FETCH_DATA: 'FETCH_DATA',
+    FETCH_DATA_SUCCESS: 'FETCH_DATA_SUCCESS',
```

Comment:
Don't keep generic names like 'FETCH_DATA_SUCCESS' make it FETCH_PROJECTDATA_SUCCESS

Reply:
Done